utils/resultwriter_test.go

```diff
@@ -92,6 +92,30 @@ func TestGetSarifTableDescription(t *testing.T) {
 			fixedVersions:       []string{},
 			expectedDescription: "| Severity Score | Direct Dependencies | Fixed Versions     |\n| :---:        |    :----:   |          :---: |\n| 3.0      | `example-package 3.0.0`       | No fix available   |",
 		},
+		{
+			name:                "Non-covered vulnerability",
+			formattedDeps:       "`example-package 3.0.0`",
+			maxCveScore:         "3.0",
+			applicable:          "Not covered",
```

Comment:
change the applicable string field into `status ApplicabilityStatus` and than at the method use `status.String()`